ntc_templates/templates/zyxel_os_cfg_nat_addr_map_get.textfsm

```diff
@@ -0,0 +1,19 @@
+Value INDEX (\d+)
+Value NAME (.+?)
+Value INTERFACE (\w+)
+Value TYPE (\S+?)
+Value LOCAL_START_IP ((\d+\.\d+\.\d+\.\d+)?)
```

Comment:
I don't think this pattern will actually work if the IPs are optional. LOCAL_START_IP will get conflated for LOCAL_END_IP as an example. If this actually exists in the real world, then would need tests, and likely another strategy to attack.

Reply:
Indeed. It should not be optional. Will correct

Reply:
I only commented on the one, but I meant it for all of them :)

Reply:
Ok, I fine tuned adding some tests with raw data I collected directly after manually configuring the address 
mapping through the UI. Let me know.
Sorry for the troubles. I am adding old work of a former colleague and I am new to networking.